cmd/gameserver/packet/character.go

```diff
@@ -11,6 +11,30 @@ import (
 	"github.com/ubis/Freya/share/rpc"
 )
 
+// NewTargetUser Packet
+func NewTargetUser(session *network.Session, reader *network.Reader) {
+	sessionId := reader.ReadUint16()
+
+	pSession := g_NetworkManager.GetSession(sessionId)
+	ctx, err := context.Parse(pSession)
+	if err != nil {
+		log.Error(err.Error())
+		return
+	}
+
+	ctx.Mutex.Lock()
```

Comment:
You should use `RLock` here, not `Lock`.

Reply:
Sorry, I typoed it)